py_hamt/store_httpx.py

```
@@ -11,6 +14,58 @@
+                if retry_at.tzinfo is None:
+                    retry_at = retry_at.replace(tzinfo=timezone.utc)
+                retry_after_seconds = (
+                    retry_at - datetime.now(timezone.utc)
+                ).total_seconds()
+            except (TypeError, ValueError, OverflowError):
+                retry_after_seconds = -1
+
+        if retry_after_seconds >= 0:
+            return min(retry_after_seconds, backoff_delay)
+
+    jitter = backoff_delay * 0.1 * (random.random() - 0.5)
```

Ruff (0.15.20)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)